packages/arb-avm-cpp/avm_values/src/value.cpp

```diff
@@ -25,6 +25,152 @@
 #include <iomanip>
 #include <ostream>
 
+Value::Value() : Value(Tuple()) {}
+Value::Value(Tuple tup) : Value(0) {
+    inner.tagged.tag = value_tuple_tag;
+    inner.tagged.inner.tuple = std::move(tup);
+}
+Value::Value(uint64_t num) : Value(uint256_t(num)) {}
+Value::Value(uint256_t num)
+    : inner{TaggedValue{value_num_tag, TaggedValueContents{num}}} {}
+Value::Value(CodePointStub code_point) : Value(0) {
+    inner.code_point = std::move(code_point);
```

Comment:
I don't think `move` does anything here since `CodePointStub` is just inline data, but can't hurt